Dockerfile

```diff
@@ -1,16 +1,13 @@
-FROM alpine:3.13.5@sha256:def822f9851ca422481ec6fee59a9966f12b351c62ccb9aca841526ffaa9f748
+FROM alpine:3.14.1@sha256:be9bdc0ef8e96dbc428dc189b31e2e3b05523d96d12ed627c37aa2936653258c
 
 USER root
 
 ENTRYPOINT ["tini", "--"]
 
-RUN ["apk", "--no-cache", "upgrade"]
-RUN ["apk", "--no-cache", "add", "tini", "dnsmasq", "bash", "curl", "openssl"]
-# naxsi and awscli are not available from the main Alpine repositories yet.
-RUN ["apk", "--no-cache", "--repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing", "add", \
-     "nginx-naxsi=1.16.1-r0", "nginx-naxsi-mod-http-naxsi=1.16.1-r0", "nginx-naxsi-mod-http-xslt-filter=1.16.1-r0"]
+RUN ["apk", "--no-cache", "-U", "upgrade"]
+RUN ["apk", "--no-cache", "add", "tini", "dnsmasq", "bash", "curl", "openssl", "python3", "py-pip", "nginx-mod-http-naxsi=1.20.1-r3"]
 
-RUN ["apk", "--no-cache", "--repository=http://dl-cdn.alpinelinux.org/alpine/v3.13/community", "add", "aws-cli=1.18.177-r0"]
+RUN ["pip", "install", "awscli"]
```

Comment:
How come we moved to installing awscli via pip instead of via apk?

Reply:
So I did try it using APK for this first, as this was how we were doing it before, but then when I tried to test it, we started getting Python "module not found" errors. Switching to using PIP to install the AWS CLI resolved this issue, and looking online, this does now seem to be the suggested way to install the AWS CLI on Alpine containers.

Reply:
Fair enough :) Sounds like it was a pain 🗡️ 

All good then :) Nice one :)